src/style.css

```diff
@@ -0,0 +1,99 @@
+:root {
+  --form-text-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc) format("truetype"),
+    url(/fonts/Avenir-Book.ttf) format("truetype"),
+    url(/fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.search {
+  margin: 20px 8px;
+  border: solid 1px #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  display: grid;
+  grid-template-columns: auto 1fr;
+  align-items: center;
+  padding: 0;
+}
+
+.icon__search {
+  height: 19px;
+  width: 19px;
+  margin: 0;
+  margin-right: 17px;
+  background-image: url(/images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.input__search {
+  border: 0 solid;
+  width: 100%;
+  font-size: 16px;
+  font-weight: 800;
+  color: var(--form-text-color);
+  /* padding: 0; */
+}
+
+.input__search:focus {
+  outline: none;
+  text-shadow: 1px 2px 2px rgba(0, 0, 0, 0.25);
+}
+
+.input__search::placeholder,
+.input__search:focus::placeholder {
+  font-family: "Avenir", sans-serif ;
```

Comment:
if I remove it here, the text in the placeholder changes to Arial. 
I thought it was happening because of this point, or?
"Remember that placeholder has its own set of styles available using ::placeholder pseudo-element."